### PR TITLE
New backend tests: CL commands and version check

### PR DESCRIPTION
**Pull request by ThePrez**

### CODE REVIEW

**Comment by ThePrez** [comment]
Also now server-side paging query test

**Review by worksofliam — COMMENTED** [review]
Check out my comments. Please also format the changed files. Thanks!

File: src/connection/sqlJob.ts [hunk 1]
```diff
@@ -121,6 +123,73 @@ export class SQLJob {
...
+    
+    return version;
+  }
+  async clcommand<T>(cmd: string): Promise<CLCommandResult> {
+    const cmdObj = {
+      id: `boop`,
+      type: `cl`,
+      cmd: cmd
+    };
+    let hmm = JSON.stringify(cmdObj);
```
[the remark this answers is not in the document]
Comment (ThePrez) [reply src/connection/sqlJob.ts]: addressed at https://github.com/halcyon-tech/vscode-db2i/pull/22/commits/b5d3554862a12e615dadd75f5d498d08c0b9ca72

File: src/connection/sqlJob.ts [hunk 2]
```diff
@@ -121,6 +123,73 @@ export class SQLJob {
...
+      id: `boop`,
+      type: `getversion`
+    };
+
+    const result = await this.send(JSON.stringify(verObj));
+
+    const version: VersionCheckResult = JSON.parse(result);
+
+    if (version.success !== true) {
+      throw new Error(version.error || `Failed to get version from packaned`);
```
[the remark this answers is not in the document]
Comment (ThePrez) [reply src/connection/sqlJob.ts]: Addressed at https://github.com/halcyon-tech/vscode-db2i/pull/22/commits/b5d3554862a12e615dadd75f5d498d08c0b9ca72

File: src/connection/sqlJob.ts [hunk 3]
```diff
@@ -121,6 +123,73 @@ export class SQLJob {
     return queryResult.data;
   }
 
+  
+  async pagingQuery<T>(correlation_id: string, sql: string, rowsToFetch?:number): Promise<QueryResult> {
```
[the remark this answers is not in the document]
Comment (ThePrez) [reply src/connection/sqlJob.ts]: Addressed at https://github.com/halcyon-tech/vscode-db2i/pull/22/commits/b5d3554862a12e615dadd75f5d498d08c0b9ca72

**Pull request merged**